tests/unit/pandas_utils/test_lusid_pandas.py

```diff
@@ -303,3 +303,112 @@ def test_response_to_df_rename_property(self):
+            identifiers={
+                "ClientInternal": "imd_35349922",
+                "Figi": "BBG000BRVH22",
+                "Isin": "GB00B1KJJ422",
+                "LusidInstrumentId": "LUID_ZJJ1IY22",
+                "Ticker": "WTB22",
+            },
+            state="Active",
+        )
+
+        test_instrument_response_2 = lusid.models.instrument.Instrument(
+            lusid_instrument_id="LUID_TEST2",
+            version=1,
+            name="Test LUID2",
+            identifiers={
+                "ClientInternal": "imd_35349900",
+                "Figi": "BBG000BRVH05",
+                "Isin": "GB00B1KJJ408",
+                "LusidInstrumentId": "LUID_ZJJ1IYJS",
+                "Ticker": "WTB",
+            },
+            state="Active",
+        )
+
+        list_response = [test_instrument_response_1, test_instrument_response_2]
+
+        instrument_df = lusid_response_to_data_frame(list_response)
+        self.assertEqual(type(instrument_df), pd.DataFrame)
+        self.assertEqual(
+            instrument_df.columns.to_list(),
+            [
+                "lusid_instrument_id",
+                "version",
+                "name",
+                "identifiers.ClientInternal",
+                "identifiers.Figi",
+                "identifiers.Isin",
+                "identifiers.LusidInstrumentId",
+                "identifiers.Ticker",
+                "state",
+            ],
+        )
+        self.assertEqual(
+            instrument_df["lusid_instrument_id"].to_list()[0], "LUID_TEST1"
+        )
+
+    def test_list_of_different_lusid_objects(self):
+
+        with self.assertRaises(TypeError) as error:
+
+            test_instrument_response_1 = lusid.models.instrument.Instrument(
+                lusid_instrument_id="LUID_TEST1",
+                version=1,
+                name="Test LUID1",
+                identifiers={
+                    "ClientInternal": "imd_35349922",
+                    "Figi": "BBG000BRVH22",
+                    "Isin": "GB00B1KJJ422",
+                    "LusidInstrumentId": "LUID_ZJJ1IY22",
+                    "Ticker": "WTB22",
+                },
+                state="Active",
+            )
+
+            test_instrument_response_2 = lusid.models.instrument.Instrument(
+                lusid_instrument_id="LUID_TEST2",
+                version=1,
+                name="Test LUID2",
+                identifiers={
+                    "ClientInternal": "imd_35349900",
+                    "Figi": "BBG000BRVH05",
+                    "Isin": "GB00B1KJJ408",
+                    "LusidInstrumentId": "LUID_ZJJ1IYJS",
+                    "Ticker": "WTB",
+                },
+                state="Active",
+            )
+
+            list_response = [
+                test_instrument_response_1,
+                test_instrument_response_2,
+                self.holding_1,
+            ]
+
+            instrument_df = lusid_response_to_data_frame(list_response)
```

Comment:
No need to assign this to a variable as it will through an exception can just write "lusid_response_to_data_frame(list_response)"